src/client/scripts/game/chess/movesets.js

```diff
@@ -26,9 +26,15 @@ const movesets = (function() {
             knights: function () {
                 return {
                     individual: [
-                        [-2,1],[-1,2],[1,2],[2,1],
-                        [-2,-1],[-1,-2],[1,-2],[2,-1]
-                    ]
+                        //[-2,1],[-1,2],[1,2],[2,1],
+                        //[-2,-1],[-1,-2],[1,-2],[2,-1]
+                    ],
+                    slideMoves: {
```

Comment:
Request to change the name of sliding moves to `sliding`, to better match `individual`!

Reply:
Vars have now been renamed